src/app/new-page/component/card/card.component.ts

```diff
@@ -0,0 +1,76 @@
+import {
+  Component,
+  Input,
+  OnChanges,
+  OnInit,
+  SimpleChanges
+} from '@angular/core';
+import { Observable } from 'rxjs';
+import { PageService } from '../../service/page-service.service';
+import { org } from '@cruglobal/godtools-shared';
+import {
+  Card,
+  Text,
+  Content
+} from 'src/app/services/xml-parser-service/xmp-parser.service';
+@Component({
+  selector: 'app-page-new-card',
+  templateUrl: './card.component.html',
+  styleUrls: ['./card.component.css']
+})
+export class CardNewComponent implements OnChanges {
+  @Input() card: Card;
+  @Input() cardIndex: number;
```

Comment:
There is a `card.position` property that would give you the position of this card so you didn't need to inject an index when creating the component.

Reply:
I don't use the input`cardIndex` anymore, as I use position. I've removed `cardIndex`.